src/lib/isMobilePhone.js

```diff
@@ -115,7 +115,9 @@ const phones = {
   'pt-BR': /^((\+?55\ ?[1-9]{2}\ ?)|(\+?55\ ?\([1-9]{2}\)\ ?)|(0[1-9]{2}\ ?)|(\([1-9]{2}\)\ ?)|([1-9]{2}\ ?))((\d{4}\-?\d{4})|(9[2-9]{1}\d{3}\-?\d{4}))$/,
   'pt-PT': /^(\+?351)?9[1236]\d{7}$/,
   'pt-AO': /^(\+244)\d{9}$/,
+  'ro-MD': /^(\+373)(\/|\s|\.|\-|\()?\d{2,3}(\/|\s|\.|\-|\))?\d{1,2}(\/|\s|\.|\-|)?\d{2}(\/|\s|\.|\-|\()?\d{2}?$/,
   'ro-RO': /^(\+?4?0)\s?7\d{2}(\/|\s|\.|\-)?\d{3}(\s|\.|\-)?\d{3}$/,
+  'ru-MD': /^(\+373)(\/|\s|\.|\-|\()?\d{2,3}(\/|\s|\.|\-|\))?\d{1,2}(\/|\s|\.|\-|)?\d{2}(\/|\s|\.|\-|\()?\d{2}$/,
```

Comment:
I'd never noticed this a alias,
could we just do `..['ru-MD'] = ..['ro-MD']`?

Reply:
It seems like you would have to have the ```..['ru-MD'] = ..['ro-MD']``` outside the definition of the validator (see [here](https://stackoverflow.com/questions/42287867/how-to-set-a-dict-value-using-another-key-of-the-same-dict)), which could be messy with it having to be assigned outside the dict.